src/vaults.lua

```diff
@@ -74,10 +86,28 @@ function vaults.vaultedTransfer(from, recipient, qty, lockLengthMs, currentTimes
 		balance = qty,
 		startTimestamp = currentTimestamp,
 		endTimestamp = currentTimestamp + lockLengthMs,
+		controller = revokable and from or nil,
 	})
 	return newVault
 end
 
+--- Revokes a vaulted transfer
+---@param from WalletAddress
```

Comment:
`from` and `recipient` are somewhat ambiguous in this use case. I don't like any of the other terms I'm thinking of better so simply adding some more context here in the descriptions might be nice.

Reply:
[2ca8dce](https://github.com/ar-io/ar-io-network-process/pull/334/commits/2ca8dcea8aef1b18d91e3aebbc920f5645474085)